tests/mocha/client/forgotPassword.coffee

```diff
@@ -0,0 +1,31 @@
+if not Meteor.isClient
+  return
+
+MochaWeb?.testOnly ->
+
+  assert = chai.assert
+
+  describe "Forgot Password", ->
+
+    @.timeout 10000
+
+    beforeEach (done) ->
+      @.timeout 10000
+      Meteor.flush()
+      LoginHelper.logout ->
+        LoginHelper.waitForVisibleForm done
+
+    it "should present forgot password link on bad password", (done) ->
+      LoginHelper.submitSignIn "apollos.person.keys@newspring.cc", "wrongPassword", ->
+        LoginHelper.waitForForgotPasswordLink done
+
+    it "should present forgot password form on click link", (done) ->
+      LoginHelper.goToForgotPassword ->
+        LoginHelper.getForgotPasswordBackButton().click()
+        done()
+
+    # it "should display password reset confirmation on submit", (done) ->
```

Comment:
this test is running sometimes. it appears that the mirror doesn't like to show the confirmation text.